rxui-kotlin/src/test/kotlin/com/artemzin/rxui/kotlin/RxUiExtensionsKtTest.kt

```diff
@@ -2,33 +2,33 @@
 
 package com.artemzin.rxui.kotlin
 
+import io.reactivex.Observable
+import io.reactivex.disposables.Disposable
+import io.reactivex.functions.Function
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.Test
 import org.mockito.Mockito.*
-import rx.Observable
-import rx.Subscription
-import rx.functions.Func1
 
 class RxUiExtensionsKtTest {
 
     // Yup, you can use whitespaces in function names in Kotlin: https://twitter.com/artem_zin/status/731151474017873920
     @Test
     fun `bind should bind ui function to Observable`() {
         // WHEN we have an Observable
-        val observable = Observable.from(listOf("a", "b", "c"))
+        val observable = Observable.fromIterable(listOf("a", "b", "c"))
 
         // AND we have a UI function
-        val uiFunc = mock(Func1::class.java) as Func1<Observable<String>, Subscription>
-        val expectedSubscription = mock(Subscription::class.java)
-        `when`(uiFunc.call(observable)).then { expectedSubscription }
+        val uiFunc = mock(io.reactivex.functions.Function::class.java) as Function<Observable<String>, Disposable>
```

Comment:
Fixed, thx

This is IDE/Kotlin issue, Kotlin has own `Function` type that does not require imports, in some places like this IDE does not even ask you for import and throws Kotlin Function type